plugins/in_winevtlog/pack.c

```diff
@@ -0,0 +1,601 @@
+/* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
```

Comment:
It might be a good idea to take the parts of this file that do basic type packing and put it in a generic file other modules can use so we can minimize code duplication.